addon-test-support/@ember/test-helpers/dom/blur.js

```diff
@@ -0,0 +1,49 @@
+import getElement from './-get-element';
+import fireEvent from './fire-event';
+import settled from '../settled';
+import isFocusable from './-is-focusable';
+import { nextTickPromise } from '../-utils';
+
+/**
+  @private
+  @method __blur__
+  @param {HTMLElement} element
+*/
+export function __blur__(element) {
```

Comment:
Why have private methods that are not async and don't return a promise? Perf?

Reply:
Basically just for reuse where they are used in another async helper already (e.g. focus is used by click).